GameOfLife/GameOfLife/Board.cs

```diff
@@ -0,0 +1,166 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    public class Board
+    {
+        /// <summary>
+        /// Two-dimensional array in which initial values are stored
+        /// and new layout created.
+        /// </summary>
+        private bool[,] initialBoard;
+
```

Comment:
no need for empty lines when there is no logical reason for it to be there, as all these variables are private